.prettierrc.js

```diff
@@ -0,0 +1 @@
+module.exports = require('@open-web3/dev-config/config/prettier.cjs');
```

Comment:
I don't even see prettier in dependencies, why is this and the other file for prettier needed?

Reply:
Removed 5ea9ef0, prettier is declared inside polkadot/dev package, as a util script, anyway not neccesary in our case for now. Iam removing actually all @open-web3 packages in another branch, and using the original polkadot/dev and polkadot/utils

Reply:
Will send a PR with those refactor in a separate way.